SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -0,0 +1,1833 @@
+	{
+		return;
+	}
+
+	USpatialActorChannel* Channel = NetDriver->GetActorChannelByEntityId(EntityId);
+	if (Channel == nullptr)
+	{
+		// If there is no actor channel as a result of the actor being dormant, then assume the actor is about to become active.
+		if (EntityHasComponent(EntityId, SpatialConstants::DORMANT_COMPONENT_ID))
+		{
+			if (AActor* Actor = Cast<AActor>(NetDriver->PackageMap->GetObjectFromEntityId(EntityId)))
+			{
+				Channel = GetOrRecreateChannelForDormantActor(Actor, EntityId);
+
+				// As we haven't removed the dormant component just yet, this might be a single replication update where the actor
+				// remains dormant. Add it back to pending dormancy so the local worker can clean up the channel. If we do process
+				// a dormant component removal later in this frame, we'll clear the channel from pending dormancy channel then.
+				NetDriver->AddPendingDormantChannel(Channel);
+			}
+			else
+			{
+				UE_LOG(LogActorSystem, Warning,
+					   TEXT("Worker: %s Dormant actor (entity: %lld) has been deleted on this worker but we have received a component "
+							"update (id: %d) from the server."),
+					   *NetDriver->Connection->GetWorkerId(), EntityId, ComponentId);
+				return;
+			}
+		}
+		else
+		{
+			UE_LOG(LogActorSystem, Log,
+				   TEXT("Worker: %s Entity: %lld Component: %d - No actor channel for update. This most likely occured due to the "
+						"component updates that are sent when authority is lost during entity deletion."),
+				   *NetDriver->Connection->GetWorkerId(), EntityId, ComponentId);
+			return;
+		}
+	}
+
+	uint32 Offset;
+	bool bFoundOffset = NetDriver->ClassInfoManager->GetOffsetByComponentId(ComponentId, Offset);
+	if (!bFoundOffset)
+	{
+		UE_LOG(LogActorSystem, Warning,
+			   TEXT("Worker: %s EntityId %d ComponentId %d - Could not find offset for component id when receiving a component update."),
+			   *NetDriver->Connection->GetWorkerId(), EntityId, ComponentId);
+		return;
+	}
+
+	UObject* TargetObject = nullptr;
+
+	if (Offset == 0)
+	{
+		TargetObject = Channel->GetActor();
+	}
+	else
+	{
+		TargetObject = NetDriver->PackageMap->GetObjectFromUnrealObjectRef(FUnrealObjectRef(EntityId, Offset)).Get();
+	}
+
+	if (TargetObject == nullptr)
+	{
+		UE_LOG(LogActorSystem, Warning, TEXT("Entity: %d Component: %d - Couldn't find target object for update"), EntityId, ComponentId);
+		return;
+	}
+
+	if (EventTracer != nullptr)
+	{
+		FSpatialGDKSpanId CauseSpanId = EventTracer->GetSpanId(EntityComponentId(EntityId, ComponentId));
+		EventTracer->TraceEvent(FSpatialTraceEventBuilder::CreateComponentUpdate(Channel->Actor, TargetObject, EntityId, ComponentId),
+								CauseSpanId.GetConstId(), 1);
+	}
+
+	ESchemaComponentType Category = NetDriver->ClassInfoManager->GetCategoryByComponentId(ComponentId);
+
+	if (Category == SCHEMA_Data || Category == SCHEMA_OwnerOnly)
+	{
+		// SCOPE_CYCLE_COUNTER(STAT_ReceiverApplyData);
```

Comment:
Ticket for adding these back in (or just add them back in, should be super simple)

Reply:
"super simple" i'll see if I can easily add these back

Reply:
added back the 5 that were here and commented out, renamed for this system